contracts/StrategyRegistry.vy

```diff
@@ -0,0 +1,70 @@
+# @version 0.2.12
+
+governance: public(address)
+pendingGovernance: public(address)
+
+event NewGovernance:
+    governance: address
+
+interface Strategy:
+    def name() -> String[64]: view
+
+struct StrategyInfo:
+    name: String[64]
+    numReleases: uint256 
+    versions: address[20]
+
+strategiesReleases: public(HashMap[bytes32, StrategyInfo])
```

Comment:
`String[64]` can be a key. Saves a hash

Reply:
Also, not really sure that using the strategy's name is all that safe, but I guess depends on who can add it and what the outcomes are if it's incorrect.

Reply:
I think it's fine, it can be based on github repo. For example:
`yearnV2-generic-lender-strat` for https://github.com/Grandthrax/yearnV2-generic-lender-strat
`SingleSidedCurve` for https://github.com/Grandthrax/SingleSidedCurve

I guess we can also normalize repo's names

Reply:
What about different versions of the vault, should the name includes the vault version?

Reply:
If we feel strongly that a string is the best domain separator to use here, then yeah this might make sense to do. The norms around "how" the string is set can be decided off-chain, and potentially evolve over time (with some discipline, of course)